lib/Wikia/src/Logger/Hooks.php

```diff
@@ -12,72 +12,6 @@
 class Hooks {
 
 	/**
-	 * Load an 'onerror' handler that forwards js errors to a log.
-	 *
-	 * @param array $vars
-	 * @param string $scripts
-	 * @return boolean
-	 *
-	 */
-	public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
-		global $wgDevelEnvironment, $wgIsGASpecialWiki, $wgEnableJavaScriptErrorLogging, $wgCacheBuster, $wgMemc;
-
-		if ( !$wgDevelEnvironment ) {
-			$onError = $wgIsGASpecialWiki || $wgEnableJavaScriptErrorLogging;
-			$key = "wikialogger-top-script-$onError";
-			$loggingJs = $wgMemc->get( $key );
-
-			if ( !$loggingJs ) {
-				$errorUrl = "//jserrorslog.wikia.com/";
-				$loggingJs = "
-					function syslogReport(priority, message, context) {
```

Comment:
This function is called in `resources/wikia/modules/log.js`. The call is "safe" as it's wrapped in an if statement, but let's remove it :)